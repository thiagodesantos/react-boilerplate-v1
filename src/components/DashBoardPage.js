import React from 'react';

const DashBoardPage = () => (
    <div>
        Pagina do dashboard
    </div>
);

export default DashBoardPage;