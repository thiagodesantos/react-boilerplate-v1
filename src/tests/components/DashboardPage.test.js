import React from 'react';
import { shallow } from 'enzyme';
import DashBoardPage from '../../components/DashBoardPage';

test('Deveria renderizar o DashBoardPage corretamente', () => {
    const wrapper = shallow( <DashBoardPage />);
    expect(wrapper).toMatchSnapshot();
});
