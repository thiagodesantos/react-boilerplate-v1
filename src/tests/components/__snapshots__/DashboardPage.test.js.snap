// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deveria renderizar o DashBoardPage corretamente 1`] = `
<div>
  Pagina do dashboard
</div>
`;
